source/_patterns/01-elements/link/link.scss

```diff
@@ -1,28 +1,34 @@
 @use "@db-ui/base/build/scss/variables" as *;
+@use "@db-ui/base/build/scss/scaling-placeholder" as *;
+@use "@db-ui/base/build/scss/color-placeholder" as *;
 @use "@db-ui/base/build/scss/helpers/functions" as *;
 @use "@db-ui/base/build/scss/icon/icons.helpers" as *;
 
 @import "../../../css/partials.meta";
-@import "link.variables";
 
 .elm-link {
-	border-radius: to-rem($pxValue: 4);
+	color: var(--db-current-color, $db-colors-neutral-on-bg-enabled);
 	display: inline-block;
+	border-radius: to-rem($pxValue: 6);
 
 	// Link underline
-	text-underline-offset: to-rem($pxValue: 3);
+	text-underline-offset: to-rem($pxValue: 1);
 	text-underline-position: under;
+	text-decoration-thickness: to-rem($pxValue: 1);
 	@supports (text-underline-offset: 3px) {
 		text-underline-position: auto;
 	}
 
-	&:active {
-		color: initial;
+	// Changing color on hover should be animated by a transition
+	transition: $db-transition-color;
+
+	&:hover {
+		color: inherit;
 	}
 
 	// "disabled" links
 	&[aria-disabled="true"] {
-		opacity: $link--disabled-opacity;
+		opacity: var(--db-disabled-text-opacity, 0.5);
```

Comment:
What about retrieving this from DB UI Base (by moving https://github.com/db-ui/core/pull/89/files#diff-3c4b77e29eed06d8c4323c5b88e5e8bd3c78b0c2e077cda82f83eef04836614fR47 to Base as well)?